app/templates/setPasswordSection/setPasswordSection.css

```diff
@@ -17,6 +17,12 @@
   line-height: 23px;
   cursor: pointer;
   color: var(--lightTextColor);
+  -webkit-touch-callout: none;
+  -webkit-user-select: none;
+  -khtml-user-select: none;
+  -moz-user-select: none;
+  -ms-user-select: none;
+  user-select: none;
```

Comment:
Our build step includes autoprefixer via 'postcss-cssnext' so we don't need the prefixed versions of `user-select` here.

Reply:
Wow, completely missed that.